docs/spec/staking/README.md

```diff
@@ -14,25 +14,6 @@ validators, ultimately determining the effective validator set for the system.
 This module will be used in the Cosmos Hub, the first Hub in the Cosmos
 network.
 
-## Contents
```

Comment:
I really liked that the old readme linked to each of the files. Can we make the new one do that too?

Reply:
k added back in... just found it a hassle to maintain, ideally we have TOC autogenerated